orderbook/src/fee.rs

```diff
@@ -257,12 +272,12 @@ impl MinFeeCalculating for MinFeeCalculator {
 
         tracing::debug!(?fee_data, ?app_data, "computing subsidized fee",);
 
-        let unsubsidized_min_fee = if let Ok(Some(past_fee)) = self
+        let unsubsidized_min_fee = if let Some(past_fee) = self
```

Comment:
I don't know if this change was intentional, but it changes the behaviour of fee estimates.

Before, if you would get a DB error reading reading past fees, then we would fall back to computing a new fee amount. Now, the whole `compute_subsidized_min_fee` call will fail. I'm fine with this behaviour change, but maybe add it to the PR descriptions?

Reply:
Right, I added this because I had an SQL error in my query before and nothing ever warned me about it. I'm ok changing this to ignoring DB errors (but we should at least print them). I felt like in other places we have made increasingly the decision to error whenever unexpected things happen (a DB being one of them). Will add to the PR description.

Reply:
To clarify, I agree with this change. Just wanted to make sure it was intentional.